### PR TITLE
Added docs about setting notifications badge through content app factory

### PR DESCRIPTION
**Pull request by dawoe**
I created a PR to make it possible to set content app notification badges through a content app factory. This will be part of 8.3

This PR is the documentation for that functionality.

Dave

### CODE REVIEW

File: Extending/Content-Apps/index.md [hunk 1]
```diff
@@ -268,3 +268,59 @@ $scope.model.badge = {
   type: "warning" // optional: determines the badge color - "warning" = dark yellow, "alert" = red, anything else = blue (matching the top-menu background color)
 };
 ```
+
+From version 8.3.0 and up it is also possible to set a notification badge form a IContentAppFactory. This is done by setting the badge property on the ContentApp model.
```
Comment (marcemarc) [inline Extending/Content-Apps/index.md]:
```suggestion
From version 8.3.0 and up it is also possible to set a notification badge from an `IContentAppFactory`. This is achieved by setting the badge property on the ContentApp model.
```

File: Extending/Content-Apps/index.md [hunk 2]
```diff
@@ -268,3 +268,59 @@ $scope.model.badge = {
...
+	            return wordCounterApp;
+			}
+
+            return null            
+        }
+    }
+}
+```
+
+Possible values for the type are Default, Alert and Warning
```
Comment (marcemarc) [inline Extending/Content-Apps/index.md]:
```suggestion
Possible values for the ContentAppBadge Type are Default, Alert and Warning
```

**Comment by dawoe** [comment]
Hi @marcemarc 

I have updated the wording. 

Dave

File: Extending/Content-Apps/index.md [hunk 3]
```diff
@@ -268,3 +268,59 @@ $scope.model.badge = {
...
+	            return wordCounterApp;
+			}
+
+            return null            
+        }
+    }
+}
+```
+
+Possible values for the ContentAppBadge Type  are Default, Alert and Warning
```
Comment (sofietoft) [inline Extending/Content-Apps/index.md]: This is great @dawoe !
Just checking in here, with a quick comment on the last line:

```suggestion
Possible values for the `ContentAppBadge` Type are *Default*, *Alert* and *Warning*.
```

**Comment by sofietoft** [comment]
Thanks @dawoe ! 😁 

@marcemarc - could you review the changes you requested here?

**Comment by dawoe** [comment]
I updated the version to 8.4.0, because it will not be released until that version

**Comment by sofietoft** [comment]
Merging this one in, as Umbraco 8.4 was released yesterday 🎉 

Thanks for the work you've made here @dawoe ! Much appreciated!

**Pull request merged**